docs/modules/operation/pages/service-assurance/monitors/FtpMonitor.adoc

```diff
@@ -1,41 +1,48 @@
 
 = FtpMonitor
```

Comment:
I think the Hint section at the very bottom probably can be removed from this document, otherwise the sentence structure needs cleanup.